test/firtool/parse-only.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: firtool %S/firtool.fir --annotation-file %S/firtool.fir.anno.json --mlir --parse-only | FileCheck %s
```

Comment:
Any reason on not to add this directly to the `firtool.fir` test? Just for clarity?

Reply:
Ah good point. Now that I think of it, that would be much smarter :wink: